Classes/ScrnS6SeekingRocket.uc

```diff
@@ -1,5 +1,42 @@
 class ScrnS6SeekingRocket extends SeekerSixSeekingRocketProjectile;
 
+//copypasta from SeekerSixSeekingRocketProjectile and added lock on to head
+simulated function Timer()
+{
+    local vector ForceDir;
+    local float VelMag;
+    local rotator HeadBoneCoords;
+    local vector LocalSeekingLocation;
+
+    if ( InitialDir == vect(0,0,0) )
+        InitialDir = Normal(Velocity);
+
+	Acceleration = vect(0,0,0);
+    Super.Timer();
+    if ( (Seeking != None) && (Seeking != Instigator) )
+    {
+        //if target has a headbone use those coords, otherwise lock onto position
+        if (Pawn(Seeking).HeadBone != '')
+        LocalSeekingLocation = Seeking.GetBoneCoords(Pawn(Seeking).HeadBone).origin; //get HeadBone name and location
```

Comment:
A dedicated server is not playing most of the animations, hence the head location on server and client will differ (that's the main reason why headshots are so buggy in KF1). This may lead to situations where projectiles are flying to the different coordinates on server and client. Especially this is bad when the projectile hits something on the way on the server side but flies through on the client.

I agree that this effect is cool, but it cannot be used in KF1 unless we introduce client-side hit detection.

Reply:
I added this only so that locked on projectiles will get double explosive damage from hitting upper chest shots, actually hitting the head wasn't intended. An initial test version seeking the Pawn.Location with Z+200 could work for large zeds but it would miss crawlers, and it wouldn't match zeds of different height as well as actually using the headbone location. So as long as using headbone coords is good enough to hit high enough to score double damage I don't mind. I think it can be added unless the headbone coords can be so far off that it makes locked rockets miss the zed entirely.